test/mat_utest.cpp

```diff
@@ -48,23 +51,21 @@ void mat_utest1()
 
     mat.inv();
 
-    assert(mat.get(0, 0) == 8);
-    assert(mat.get(0, 1) == 6);
-    assert(mat.get(0, 2) == 1);
-    assert(mat.get(1, 0) == 7);
-    assert(mat.get(1, 1) == 9);
-    assert(mat.get(1, 2) == 10);
-    assert(mat.get(2, 0) == 0);
-    assert(mat.get(2, 1) == 6);
-    assert(mat.get(2, 2) == 5);
+    ASSERT_EQ(mat.get(0, 0), 8);
+    ASSERT_EQ(mat.get(0, 1), 6);
+    ASSERT_EQ(mat.get(0, 2), 1);
+    ASSERT_EQ(mat.get(1, 0), 7);
+    ASSERT_EQ(mat.get(1, 1), 9);
+    ASSERT_EQ(mat.get(1, 2), 10);
+    ASSERT_EQ(mat.get(2, 0), 0);
+    ASSERT_EQ(mat.get(2, 1), 6);
+    ASSERT_EQ(mat.get(2, 2), 5);
 }
 
-void mat_utest2()
+TEST(MatrixTest, Test2) // NOLINT
```

Comment:
@lamphamsy If you have a better name than Test2, I'll be glad to rename it (it was named utest2 before so I kept it like that)

Reply:
Done